core/src/include/ModelArrayRef.hpp

```diff
@@ -1,45 +1,49 @@
 /*!
  * @file ModelArrayRef.hpp
  *
- * @date Apr 20, 2022
+ * @date 22 Feb 2023
```

Comment:
Wondering if we can automate a check that ensures we update these as they are changed? I've used proprietary tools that automate this on all touched files but not tried with any public ones. Didn't realise the standard for us was to update these --- I prefer it, but don't recall reading that on the repo readme.

Reply:
They probably should be updated, but haven't been.